android/aepsdk-bom/sdk-bom/build.gradle

```diff
@@ -42,6 +42,7 @@ dependencies {
     implementation 'com.adobe.marketing.mobile:edgeidentity:latest.release'
     implementation 'com.adobe.marketing.mobile:edgeconsent:latest.release'
     implementation 'com.adobe.marketing.mobile:edgebridge:latest.release'
+    implementation 'com.adobe.marketing.mobile:edgemedia:latest.release'
```

Comment:
do we need to update bom.properties too when you add a new extension or are those generated automatically?